### DISCUSSION TITLE
Пример смартапа с монетизацией на Code

### DISCUSSION THREAD

**Discussion by snyuryev** [sberdevices/salute-issues#189] [SmartMarket Studio]
Было бы здорово посмотреть на шаблон (пример) смартапа с монетизацией на Code.

**Comment by Genenko-Svetlana** [comment]
@skobets

**Reply by subnetsRU** [reply]
Вроде бы уже сделали. Разве нет? Есть же уже шаблон "Пример подключения монетизации"

**Reply by snyuryev** [reply]
@subnetsRU да, теперь уже есть
